packages/fxa-auth-server/lib/error.js

```diff
@@ -1456,5 +1458,52 @@ function scrubHeaders(headers) {
   return scrubbed;
 }
 
+function appErrorFromOauthError(err) {
```

Comment:
This function previously live here: https://github.com/mozilla/fxa/blob/train-195/packages/fxa-auth-server/lib/oauthdb/utils.js#L15

Reply:
I noticed the old one also logged an [`oauthdb.mapOAuthError` message](https://github.com/mozilla/fxa/blob/train-195/packages/fxa-auth-server/lib/oauthdb/utils.js#L68-L72) in the default case where the error's not found in the translation list. Did we ever look at those error logs in the past? Would it be useful to add a new logger call in the default case here?

Reply:
The switch cases are exhaustive as of today, but if we did add new oauth errnos in the future (which we shouldn't, OauthError should be deprecated and we should only be making new AppErrors) the default ought to be a pass-through, not an `unexpectedError`. So I'll change that default case.